.gitignore

```diff
@@ -12,6 +12,7 @@
 *.ihex
 *.pyc
 *~
+*.vscode
```

Comment:
The recommended way to deal with this is through your local `.git/info/exclude`. The top-level `.gitignore` is for files directly associated to the project (e.g. output binaries, temporary files generated by the build system)

Reply:
Reverted changes to gitignore as requested